pkg/util/clusteroperator/status_manager.go

```diff
@@ -85,8 +85,8 @@ func (status *StatusManager) SetConfigSuccess() error {
 	status.configFailure = false
 	return status.Set(
 		&configv1.ClusterOperatorStatusCondition{
-			Type:    configv1.OperatorFailing,
-			Status:  configv1.ConditionFalse,
+			Type:   configv1.OperatorFailing,
+			Status: configv1.ConditionFalse,
```

Comment:
(this is also unrelated and yeah, we need a "verify" script/target)

Reply:
We totally have one, and I'm not sure why it's been passing.

Reply:
Ah, because it always exits 0.